api/TrustedHTML.json

```
@@ -46,8 +48,59 @@
           "deprecated": false
         }
       },
+      "toJSON": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TrustedHTML/toJSON",
+          "spec_url": "https://w3c.github.io/webappsec-trusted-types/dist/spec/#dom-trustedhtml-tojson",
+          "support": {
+            "chrome": {
+              "version_added": "90"
+            },
+            "chrome_android": {
+              "version_added": "90"
+            },
+            "edge": {
+              "version_added": null
```

```suggestion
              "version_added": "90"
```

Just mirroring Edge is typically safe for new APIs, but also confirmed by testing Edge 89 and 90 on https://mdn-bcd-collector.appspot.com/tests/api/TrustedHTML/toJSON.